tests/rkt_seccomp_test.go

```diff
@@ -22,27 +22,13 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/rkt/rkt/common"
```

Comment:
commit message:

"However we **are** testing that blocking"...

(yes, I know it's my own commit message :joy:)